src/analyses/natural_loops.h

```diff
@@ -29,23 +29,23 @@ class natural_loops_templatet
 
   loop_mapt loop_map;
 
-  inline void operator()(P &program)
+  void operator()(P &program)
   {
     compute(program);
   }
 
   void output(std::ostream &) const;
 
-  inline const cfg_dominators_templatet<P, T, false>& get_dominator_info() const
+  const cfg_dominators_templatet<P, T, false>& get_dominator_info() const
```

Comment:
I'm going to add one commit to this PR that addresses all of those (736 across the code base).